spec/models/claim_spec.rb

```diff
@@ -484,14 +502,23 @@
 
   describe "#decision" do
     it "returns the latest decision on a claim" do
-      claim = build(:claim, :submitted)
-      claim.decisions.append(build(:decision, result: "approved", created_at: 7.days.ago))
-      claim.decisions.append(build(:decision, result: "rejected", created_at: DateTime.now))
+      claim = create(:claim, :submitted)
+      create(:decision, result: "approved", claim: claim, created_at: 7.days.ago)
+      create(:decision, result: "rejected", claim: claim, created_at: DateTime.now)
 
       expect(claim.decision.result).to eq "rejected"
     end
   end
 
+  describe "#decision" do
```

Comment:
We already describe `#decision` above this, could we move this example to that block?